### DISCUSSION TITLE
Chip identification / replacement

### DISCUSSION THREAD

**Discussion by dummbold** [ct-Open-Source/tuya-convert#1144] [Q&A]
Hi,

I am really done with this Tuya stuff! Bought a LSC smart plug at Action and installed it. Plugged my freezer in to measure the power consumption. The app switched off the socket itself without me doing anything – and all my frozen goods are gone now! CRAP!

Now i will try to tasmotize it – but how can I indentify the chip?

Here's all I can find about the socket:

-------------------------------

Brandname: LSC
EAN Code: 8712879159025
Product Code: 3202087 
Factory model number: SPL-W-TY-PM-EU-RY-L

On the backside (bottom) there are other numbers:

3202087
AL D 24703

On the Chip is a printing saying:

2403
100

And a label with theese two numbers:

654029
249495A1

The QR-Code on the labes is:

249495A10BA30

------------------------------------------

Is it possible to identify the chip from this information?
ESP82xx chips with Tasmota are available for a small price. Is it possible just to replace the whole Chip instead of spending hours with tasmotizing ... that doesn't work at the end?

(I also have some 10 LSC lamps and bulbs and would prefer to go this way)

**Comment by kueblc** [comment]
I feel your pain, I've heard from others who have had these plugs turn on or off with no clear reason.

You mention the print on the chip, so you have the device open already? Could you share a photo?

**Comment by dummbold** [comment]
Opening was a bit tricky as the clips are in the edges. But it I finally made it with two small screwdrivers and a guitar plektrum.
Please see ebay offer (last link). This one looks good for me - at least the 6 pins on the front seem to be identical and the shape seems good to.

![IMG_5699](https://github.com/user-attachments/assets/075f1cbb-fa48-4382-be5c-5f9ea7409ff8)
![IMG_5698](https://github.com/user-attachments/assets/6e39c508-6970-41b4-a0ed-82ae260b7554)

https://www.ebay.de/itm/375265674047?mkcid=16&mkevt=1&mkrid=707-127634-2357-0&ssspo=bqgjqyb0tla&sssrc=2047675&ssuid=&widget_ver=artemis&media=COPY

**Comment by kueblc** [comment]
From the backside of the blue board, it doesn't look like any ESP I recognize, but difficult to say with certainty without seeing the front. If it was, on the other side you would see either a small square IC with the Espressif logo, or a metal cap clearly stating what board it is.

It does look like the board you linked from eBay should be an easy swap, but I make no warranty. Maybe others can chime in with their experience replacing the board.

Depending on what board is in there now, you might have luck with [tuya-cloudcutter](https://github.com/tuya-cloudcutter/tuya-cloudcutter/) and OpenBeken instead of Tasmota.

**Comment by FerFemNemBem** [comment]
Hi,

this is a recent plug (2024-03). Most likely it already contains a Beken 
BK7231N.
OpenBeken would be a tasmota like alternative for this plug.

Some more details and a HowTo ca be found here: 
https://www.elektroda.com/rtvforum/topic4087228.html

Cheers,
FFNB

Am 27.11.2024 um 21:42 schrieb dummbold:

**Comment by dummbold** [comment]
Okay, i finally got it out with desoldering pump and wick – but I lost a contact.

![IMG_5703](https://github.com/user-attachments/assets/6159fd94-aad5-4959-8ede-a882ab01bf24)
![IMG_5705](https://github.com/user-attachments/assets/fa93da8d-a501-4662-ac41-26d94220a394)

So just for me to know and the community: is this a Beken? Can I replace it with the Chip from the attached ebay offer? 
(2€ for a new module is much cheaper then all the work with de-beken-ing)

![Bildschirmfoto 2024-11-29 um 14 01 33](https://github.com/user-attachments/assets/204077f8-5773-4152-8d17-013cf93a8798)

**Comment by kueblc** [comment]
This is the [Tuya CB2S](https://developer.tuya.com/en/docs/Document/cb2s-module-datasheet?id=Kamwxuk0z7j9x), which is indeed powered by a Beken BK7231N.

I was also able to find this device on the [tuya-cloudcutter device list](https://github.com/tuya-cloudcutter/tuya-cloudcutter.github.io/blob/master/devices/lsc-3202087-plug-with-energy-monitoring-v1.3.5.json). You should be able to install OpenBeken on this device OTA.

**Comment by dummbold** [comment]
Thanks … and I just noticed, that Tasmota is NOT installed on the ebay chips. There‘s a text in a bad translation that makes me assume that one has to flash the chip itself. So i still need to do some work which is not easy if you only use and own Macs!

**Comment by dummbold** [comment]
Does anybody know if there is a plug/socket/adapter out there for these modules that I can solder on a little circuit board together with the USB/TTL converter? So it should not be necessary to solder them twice (for flashing and installation on the board. I have about ten of these guys ...
